src/picongpu/include/particles/pusher/particlePusherBoris.hpp

```diff
@@ -36,8 +36,8 @@ struct Push
     template<typename T_Efield, typename T_Bfield, typename T_Pos, typename T_Mom, typename T_Mass,
              typename T_Charge, typename T_Weighting>
         __host__ DINLINE void operator()(
-                                            const T_Bfield bField,
-                                            const T_Efield eField,
+                                            const T_Bfield functorBField,
```

Comment:
Please rename `T_Bfield` to `T_FunctorBField` to avoid that the user misunderstand it as type of field B.